src/azure-cli/azure/cli/command_modules/acs/_params.py

```diff
@@ -440,6 +462,39 @@ def load_arguments(self, _):
         c.argument('acr', validator=validate_registry_name)
         c.argument('node_name')
 
+    with self.argument_context('aks maintenanceconfiguration') as c:
+        c.argument('cluster_name', help='The cluster name.')
+
+    for scope in ['aks maintenanceconfiguration add', 'aks maintenanceconfiguration update']:
+        with self.argument_context(scope) as c:
+            c.argument('config_name', options_list=[
+                       '--name', '-n'], help='The config name.')
+            c.argument('config_file', help='The config json file.')
+            c.argument('weekday', help='Weekday on which maintenance can happen. e.g. Monday')
+            c.argument('start_hour', type=int, help='Maintenance start hour of 1 hour window on the weekday. e.g. 1 means 1:00am - 2:00am')
+            c.argument('schedule_type', arg_type=get_enum_type(schedule_types),
+                       help='Schedule type for non-default maintenance configuration.')
+            c.argument('interval_days', type=int, help='The number of days between each set of occurrences for Daily schedule.')
+            c.argument('interval_weeks', type=int, help='The number of weeks between each set of occurrences for Weekly schedule.')
+            c.argument('interval_months', type=int, help='The number of months between each set of occurrences for AbsoluteMonthly or RelativeMonthly schedule.')
+            c.argument('day_of_week', help='Specify on which day of the week the maintenance occurs for Weekly or RelativeMonthly schedule.')
+            c.argument('day_of_month', help='Specify on which date of the month the maintenance occurs for AbsoluteMonthly schedule.')
```

Comment:
Please explain the format of these two parameters need to be passed in in the help message.